.gitignore

```diff
@@ -11,6 +11,9 @@
 # Ignore the user's VSCode settings.
 /.vscode/settings.json
 
+# Ignore the user's Idea settings.
+/.idea/
```

Comment:
This would be fine to add separately, it probably shouldn't be mixed with an implementation PR.